src/Support/ResolveInfoFieldsAndArguments.php

```diff
@@ -132,15 +143,67 @@ private function foldSelectionSet(SelectionSetNode $selectionSet, int $descend):
         return $fields;
     }
 
-    private function getValue(ArgumentNode $argumentNode)
+    /**
+     * @param ValueNode $value
+     * @return mixed
+     * @throws \Exception
+     */
+    private function getValue(ValueNode $value)
     {
-        $value = $argumentNode->value;
         if ($value instanceof VariableNode) {
             $variableName = $value->name->value;
 
             return $this->info->variableValues[$variableName] ?? null;
         }
+        // Scalar Types
+        if ($value instanceof IntValueNode || $value instanceof FloatValueNode || $value instanceof StringValueNode || $value instanceof BooleanValueNode || $value instanceof EnumValueNode) {
```

Comment:
Please break the lines: there are so many ORs, they really need to be better readable.

For simplicity, I suggest each one gets its own line:
```php
        if ($value instanceof IntValueNode ||
            $value instanceof FloatValueNode ||
…
```